src/components/edit/thematic/ThematicDialog.js

```diff
@@ -259,40 +259,41 @@ export class ThematicDialog extends Component {
                                     errorText={indicatorError}
                                 />,
                             ]}
-                            {valueType === 'de' && [
-                                // Data element
-                                <DataElementGroupSelect
-                                    key="group"
-                                    dataElementGroup={dataElementGroup}
-                                    onChange={setDataElementGroup}
-                                    style={styles.select}
-                                />,
-                                dataElementGroup && (
-                                    <TotalsDetailsSelect
-                                        key="totals"
-                                        operand={operand}
-                                        onChange={setOperand}
-                                        style={styles.select}
-                                    />
-                                ),
-                                operand === true ? (
-                                    <DataElementOperandSelect
-                                        key="element"
+                            {valueType === 'de' ||
```

Comment:
`valueType === 'de' || ...` is unnecessary here since we're immediately checking `valueType === 'dc'` right afterwards